docs/content/5.navigation/1.vertical-navigation.md

```diff
@@ -53,6 +53,241 @@ const links = [{
 ```
 ::
 
+### Customize UI
```

Comment:
This section already lives here https://ui.nuxtlabs.com/getting-started/examples#verticalnavigation

Reply:
Yes, but I think I should also add it here, or add a link?

Reply:
A `:callout` component might be more appropriate I guess.